### ISSUE TITLE
piezo in blutooth firmware makes a strange sound

### ISSUE THREAD

**Issue by nathanvale** [Makeblock-official/mbot_nodebots#10]
When using usb i can hear the tune from the example but when using bluetooth just a weird noise.

**Comment by ajfisher** [comment]
Yeah this is probably a timing issue as the piezo commands need to go **fast** to get the right frequency. I will give this a go later and see if I can improve the speed at all.

**Comment by lynnaloo** [comment]
I just ran into this same issue. Any luck on this or anything I can do to help?

**Comment by lynnaloo** [comment]
I spent some time going down the path of speeding up the BT module, but I never got anything more than the high-pitched beeping.

**Comment by ajfisher** [comment]
Yeah I think this is a running coordination issue with BT and a blocking
firmata process. I imagine we could build a buffer to queue the notes then
send a play message maybe...
